Makefile

```diff
@@ -252,4 +252,4 @@ version-docs:
 		-w /docs \
 		-u $(shell id -u):$(shell id -g) \
 		node:${NODE_VERSION} \
-		sh -c "yarn install --frozen-lockfile && yarn run docusaurus docs:version ${NEWVERSION}"
```

Comment:
looks like `preset-classic` got stuck on 2.0.1. can we update to `2.1.0` https://github.com/Azure/eraser/blob/main/docs/package.json#L18

and then run `yarn` again (which should update the lockfile) and let's re-add the `--frozen-lockfile`